backend/server/src/test/java/ai/verta/modeldb/BranchTest.java

```
@@ -483,9 +486,10 @@ public void createGetListDeleteBranchTest() {
         Assert.fail();
       } catch (StatusRuntimeException e) {
         Assert.assertEquals(Code.NOT_FOUND, e.getStatus().getCode());
-        e.printStackTrace();
       }
 
+    } catch (Exception ex) {
+      ex.printStackTrace();
```

Should this be a throw instead of a print?

---

this should be not there. I will remove it.